core/src/main/java/com/sequenceiq/cloudbreak/service/environment/EnvironmentService.java

```diff
@@ -21,7 +21,8 @@ public void checkEnvironmentStatus(Stack stack, Set<EnvironmentStatus> desiredSt
         if (stack.getEnvironmentCrn() != null) {
             DetailedEnvironmentResponse environmentResponse = environmentClientService.getByCrn(stack.getEnvironmentCrn());
             if (!desiredStatuses.contains(environmentResponse.getEnvironmentStatus())) {
-                throw new BadRequestException("Environment should be available, but the status is " + environmentResponse.getEnvironmentStatus());
+                throw new BadRequestException("This action is require to Environment be available, but the status is "
```

Comment:
pls rewrite like this: "This action requires the Environment to be available"

Reply:
👍  thanks